Makefile

```diff
@@ -16,10 +18,24 @@ cook-image:
 push-image:
 	docker push $(DOCKER_TAG)
 
-.PHONY: test
-test:
-	tox
+venv: requirements.txt
+	python ./vendor/venv-update venv= venv -ppython3 install= -r requirements.txt
+
+.PHONY: install-hooks
+install-hooks: venv
+	$(BIN)/pre-commit install -f --install-hooks
+
+test: lint
+
+.PHONY: lint
+lint: venv
+	$(BIN)/pre-commit run --all-files
 
 .PHONY: clean
 clean:
-	rm -r .tox
+	rm -rf venv
+
+.PHONY: update-requirements
+update-requirements: venv
+	$(BIN)/upgrade-requirements
+	sed -i 's/^ocflib==.*/ocflib/' requirements.txt
```

Comment:
You shouldn't need this since we don't even use ocflib in this project. This is just here to unpin ocflib since we don't want to pin it since we release all the time and usually don't have anything incompatible to worry about on that front.